app/src/components/mode-specific/desktop/InterceptTraffic/WebTraffic/TrafficTableV2/index.jsx

```diff
@@ -513,10 +514,18 @@ const CurrentTrafficTable = ({
 
   const getApplogAvatar = useCallback(
     (key, logName) => {
-      const logNameURI = decodeURIComponent(logName.trim());
-      const avatarDomain = APPNAMES[logNameURI.split(" ")[0].toLowerCase()];
-      const avatarUrl = `https://www.google.com/s2/favicons?domain=${avatarDomain}`;
-      return getLogAvatar(key, logNameURI, avatarUrl);
+      try {
+        const logNameURI = decodeURIComponent(logName.trim());
+        const avatarDomain = APPNAMES[logNameURI.split(" ")[0].toLowerCase()];
+        const avatarUrl = `https://www.google.com/s2/favicons?domain=${avatarDomain}`;
+        return getLogAvatar(key, logNameURI, avatarUrl);
+      } catch (e) {
+        console.log("faulty logname: ", logName);
+        captureException(e);
+        const avatarUrl = "https://www.google.com/s2/favicons?domain=randooooom.com"; // random url to show the unfound icon
+        const name = `unknown-${key}`;
+        return getLogAvatar(key, name, avatarUrl);
+      }
```

Comment:
- Uses sentry.captureMessage instead of exception
- Moved to finally instead of exception
```suggestion
      try {
          const logNameURI = decodeURIComponent(logName.trim());
       } catch (e) {
           Logger.log("faulty logname: ", logName);
           Sentry.captureMessage(e);
      } finally {
          const avatarDomain = APPNAMES[logNameURI.split(" ")[0].toLowerCase()];
          const avatarUrl = `https://www.google.com/s2/favicons?domain=${avatarDomain}`;
          return getLogAvatar(key, logNameURI, avatarUrl);
      }
```